### DISCUSSION TITLE
How about having ARCHITECTURE.md?

### DISCUSSION THREAD

**Discussion by amitsavani** [chatwoot/chatwoot#1740] [Q&A]
There is a good worthy discussion going on at [Hacker News](https://news.ycombinator.com/item?id=26048784) why a large opensource project like this should have an architecture diagram.

I was just looking around how to plugin more subscribers to account creation even and I am still struggling to understand the puzzle. If there would have been an architecture diagram, then it would be easier to understand such puzzles.

**Comment by pranavrajs** [comment]
@amitsavani Thanks for the suggestion, [this doc](https://github.com/chatwoot/chatwoot/wiki/Building-on-Top-of-Chatwoot:-Importing-Existing-Contacts-and-Creating-Conversations) will be a start. We can add more in the future.
